css-transitions/styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: background-color 0.5s ease-in-out;
+}
+
+.card-container {
+  text-align: center;
+}
+
+.card-container:last-child {
+  transition: background-color 0.5s ease-in-out;
+}
+
+.custom-button:hover {
+  background-color: purple;
+  background-color: red;
+}
+
+.card-container:first-child {
+  transition: 0.5s ease-in-out 0.5s;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+  transition: 0.5s ease-in-out;
+}
+
+.card:hover {
+  transform: scale(2.2);
+  transform: scale(1.1);
```

Comment:
This is a duplicate of the same property, which is not supposed to be done. The bottom one is the one being used here, you can just remove the top one.